WebHostLib/generate.py

```diff
@@ -167,9 +175,11 @@ def task():
                 if gen is not None:
                     gen.state = STATE_ERROR
                     meta = json.loads(gen.meta)
-                    meta["error"] = (
-                            "Allowed time for Generation exceeded, please consider generating locally instead. " +
-                            e.__class__.__name__ + ": " + str(e))
+                    meta["error"] = (f"Allowed time for Generation exceeded, "
+                                     f"please consider generating locally instead. "
+                                     f"{format_exception(e)}")
```

Comment:
went a little too hard on the fstrings now lol, fstrings with plain data are worse than normal strings with the same data,
also I don't think `f"{var}"` is very helpful, so something like this would probably be best
```suggestion
                    meta["error"] = ("Allowed time for Generation exceeded, " +
                                     "please consider generating locally instead. " +
                                     format_exception(e))
```
(multiple ways to format this though depending on what indenting to use etc. obviously)